beacon_node/lighthouse_network/Cargo.toml

```diff
@@ -38,10 +38,11 @@ directory = { path = "../../common/directory" }
 regex = "1.3.9"
 strum = { version = "0.21.0", features = ["derive"] }
 superstruct = "0.4.0"
-open-metrics-client = "0.14.0"
+prometheus-client = "0.15.0"
 
 [dependencies.libp2p]
-version = "0.42.1"
+git = "https://github.com/sigp/rust-libp2p"
+branch = "libp2p-gossipsub-interval-hotfix"
```

Comment:
We should tag the specific commit. 

I've been in trouble before where we've added to the branch or someone reverts back to here but that branch is different or cargo update got a different commit. I've learnt to use specific commits so we can't get in trouble here.

Reply:
done!